blog.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="zxx"> 
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>A Dopt A Dog</title>
+	<link rel="stylesheet" type="text/css" href="styles/adopt-dog.css">
+  </head>
+  
+<body>
+<header>
```

Comment:
You'll want to indent everything in the body in by one level. A minor thing, but good practice.